SharePointFramework/PortfolioWebParts/src/components/ProjectTimeline/index.tsx

```diff
@@ -289,10 +320,48 @@ export class ProjectTimeline extends Component<IProjectTimelineProps, IProjectTi
         itemProps: { style },
         project: project.title,
         projectUrl: project.url,
-        phase: project.phase
+        phase: project.phase,
+        type: project.type,
+        budgetTotal: project.budgetTotal,
+        costsTotal: project.costsTotal
+      } as ITimelineItem
+    })
+
+    const phases: ITimelineItem[] = timelineItems.filter((item) => item.type !== strings.ProjectLabel).map((item, id) => {
```

Comment:
What's the filter doing?

Reply:
I see that this is no longer needed, will remove it. Thanks for noticing!